frontend/src/pages/ResetPasswordPage.js

```diff
@@ -102,13 +103,15 @@ export default function ResetPasswordPage({ user }) {
 									</Grid>
 									{errorMessage !== "" && <Grid item xs={12}><Alert severity="error">{errorMessage}</Alert></Grid>}
```

Comment:
Is it feasible to put this error message in its own function? Because I see this over and over again. Then it will be easier to change if needed and make git diffs smaller.

Reply:
I like this idea. I will implement this in my next PR.